### DISCUSSION TITLE
[Feature]: add `Deno` support

### DISCUSSION THREAD

**Discussion by hemedani** [web-infra-dev/rsbuild#3300] [General]
### What problem does this feature solve?

just support `Deno`

### What does the proposed API look like?

I'm not any idea about it

**Comment by chenjiahan** [comment]
Could you describe your usage scenario?

**Comment by shangzx-top** [comment]
I want to use esbuild in Deno in a more 'pure' way, eliminating the need for npm or Node.js-specific syntax—at least during development. This is coming from a slightly obsessive JS coder.
